hawkular-inventory-api/src/main/java/org/hawkular/inventory/base/BasePreCommit.java

```diff
@@ -534,7 +537,7 @@ boolean canBeIdentityRoot() {
         void loadFrom(Transaction<BE> tx) throws ElementNotFoundException {
             if (element == null || loadingTx != tx) {
                 loadingTx = tx;
-                Discriminator disc = Discriminator.time(Instant.now());
+                Discriminator disc = Discriminator.time(/*isDelete ? txStart :*/ Instant.now());
```

Comment:
This is a leftover from a wrong logic I put in place at some point during the impl. I should revert that...